### DISCUSSION TITLE
display on a Pixel screen with ASP32 card?

### DISCUSSION THREAD

**Discussion by Odroid13** [j-kaltes/Juggluco#149] [Ideas]
I would like to know if it is possible to develop software for an ESP32 card in order to display blood glucose measurements on an LED matrix such as Tidbyt for example.  that would be really great.  The goal is to do the same with Tidbyt and the Nightscout software but at a lower cost, I ordered 2 ESP32 cards with 64x64 and 64x32 screens for less than 37 euros.  For those who don't know Tidbyt is their display here is the link to a video: https://youtu.be/Lm6CO424iT4

**Comment by j-kaltes** [comment]
If it can use TCP/IP (via WIFI or so), you can use the [web server](https://www.juggluco.nl/Juggluco/webserver.html) in Juggluco to receive glucose values.

**Reply by Odroid13** [reply]
Thank you for your response, I will look at the documentation.  If I can get the display on ESP32 it would be great for my 3 year old daughter.
